src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/communityapi/model/CommunityApiCourtAppearanceResponse.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.justice.probation.courtcaseservice.restclient.communityapi.model;
+
+
+import java.time.LocalDateTime;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.annotations.ApiModel;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import uk.gov.justice.probation.courtcaseservice.service.model.KeyValue;
+
+@ApiModel(description = "Court Appearances from Community API")
```

Comment:
Won't be shown in Swagger so I think we can remove this?
```suggestion
```

Reply:
Oh yes !